src/modules/userDashboard/components/YourReferrals.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import CommonCard from '@/components/elements/CommonCard';
+import NotFound from '@/components/pages/NotFound';
+import useAuth from '@/modules/auth/hooks/useAuth';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import UserStepCard from '@/modules/userDashboard/components/UserStepCard';
+import {
+  GetUserCeAssignedStepsDocument,
+  GetUserCeAssignedStepsQuery,
+  GetUserCeAssignedStepsQueryVariables,
+  UserCeReferralStepFieldsFragment,
+} from '@/types/gqlTypes';
+
+const YourReferrals = () => {
+  const { user } = useAuth();
+  if (!user) return <NotFound />;
+
+  return (
+    <>
+      <CommonCard title='Referrals' sx={{ pb: 1 }}>
+        <GenericTableWithData<
+          GetUserCeAssignedStepsQuery,
+          GetUserCeAssignedStepsQueryVariables,
+          UserCeReferralStepFieldsFragment
+        >
+          queryVariables={{}}
+          queryDocument={GetUserCeAssignedStepsDocument}
+          columns={[]}
+          pagePath='userDashboard.ceReferralSteps'
+          noData='No referral tasks assigned to you'
```

Comment:
I'm seeing a border under this 🤔  it should probably be fixed in the underlying table component, or as a workaround:
`noData={<>{'No referral tasks assigned to you'}</>}`

Reply:
Hm, does that fix it for you? I'm still seeing a border either way. Thanks for pointing that out!
![Screenshot 2025-05-28 at 2 07 37 PM](https://github.com/user-attachments/assets/39046007-afee-4a23-bd2e-877a6e172fcf)
Something like a `dontWrapNoData` prop seems messy/annoying, but maybe correct to add. I'm worried that modifying `GenericTable` to treat a `ReactNode` differently from a `string` might have unanticipated knock-on effects on other tables.

Reply:
I think I will propose `noDataCellStyles` as a new prop

Reply:
Hmm okay. The "no data" display has gotten zero love. I think it could potentially be fine to just ensure there is no bottom border directly [here](https://github.com/greenriver/hmis-frontend/blob/a766573c0d239115a293a20b05da1e0fcbbdd959/src/components/elements/table/GenericTable.tsx#L421)? I don't see why we would want a border on the "no data" message. If the table is occurring inside a bordered card, it doesn't make a difference anyway.